homeassistant/components/bosch_shc/config_flow.py

```diff
@@ -143,11 +150,15 @@ async def async_step_credentials(
         errors: dict[str, str] = {}
         if user_input is not None:
             zeroconf_instance = await zeroconf.async_get_instance(self.hass)
+            # unique_id uniquely identifies the registered controller and is used
+            # to save the key/certificate pair for each controller separately
+            unique_id = self.info.get("unique_id") or ""
```

Comment:
why do we use `.get` here? Previously we just used `[""]`

Reply:
Thank you for the review.
I used `.get()` due to it being a personal preference as it does not assume the key to be present. But as you pointed out that was the pattern used before, so I changed it back to `[""]`.

Just as a side note: The `or ""` ensures that `unique_id` is of type `str` and not `None`. We could instead also change the signature of `create_credentials_and_validate` and handle `None` values there  if that is the preferred way.